wgpu-core/Cargo.toml

```diff
@@ -39,19 +39,22 @@ gpu-descriptor = { git = "https://github.com/zakarumych/gpu-descriptor", rev = "
 hal = { package = "gfx-hal", git = "https://github.com/gfx-rs/gfx", rev = "0244e3401e9f127617cb8636397048584e7bfe8a" }
 gfx-backend-empty = { git = "https://github.com/gfx-rs/gfx", rev = "0244e3401e9f127617cb8636397048584e7bfe8a" }
 
-[target.'cfg(all(unix, not(target_os = "ios"), not(target_os = "macos")))'.dependencies]
+[target.'cfg(all(not(target_arch = "wasm32"), all(unix, not(target_os = "ios"), not(target_os = "macos"))))'.dependencies]
```

Comment:
it's already `all(a, b)`, so we should just make it `all(a, b, c)` instead of wrapping everything into another `all`

Reply:
My goal is to make this more readable by having the exact same pattern for the different backend dependency declarations.
Here again, let me know if you still want me to merge the two `all()`.